WindowsServerDocs/administration/performance-tuning/hardware/index.md

```diff
@@ -43,7 +43,7 @@ When your computer runs low on memory and it needs more immediately, Windows use
 You can optimize paging by using the following guidelines for page file placement:
 - Isolate the page file on its own storage device, or at least make sure it doesn't share the same storage devices as other frequently accessed files. For example, place the page file and operating system files on separate physical disk drives.
 
-- Place the page file on a drive that is not fault-tolerant. If the disk fails, a system crash is likely to occur. If you place the page file on a fault-tolerant drive, remember that fault-tolerant systems are often slower to write data because they write data to multiple locations.
+- Place the page file on a drive that is fault-tolerant. If the disk fails, a system crash is likely to occur. If you place the page file on a fault-tolerant drive, remember that fault-tolerant systems are often slower to write data because they write data to multiple locations.
```

Comment:
Based on your suggested change, I propose the following additional change:
```suggestion
- Place the page file on a drive that is fault-tolerant. If a non-fault-tolerant disk fails, a system crash is likely to occur. If you place the page file on a fault-tolerant drive, remember that fault-tolerant systems are often slower to write data because they write data to multiple locations.
```